src/internal/storage/chunk/storage.go

```diff
@@ -19,7 +19,7 @@ const (
 	TrackerPrefix        = "chunk/"
 	prefix               = "chunk"
 	defaultChunkTTL      = 30 * time.Minute
-	defaultPrefetchLimit = 10
+	DefaultPrefetchLimit = 10
```

Comment:
It seems like this const is only used below in `NewStorage()` is it exported because the plan is to reuse it in other packages in future PRs?

Reply:
Yeah, I want to eventually merge the configurations for prefetching in the file set and chunk layer.